distribution/resources/bin/gateway.bat

```diff
@@ -132,14 +132,10 @@ REM Start the gateway using internal ballerina distribution as the runtime
     IF %ERRORLEVEL% NEQ 0 (
         ECHO WARN: Can't find powershell in the system!
         ECHO WARN: STDERR and STDOUT will be piped to %GW_HOME%\logs\microgateway.log
-        SET JAVA_ARGS=-Xms%JAVA_XMS_VALUE% -Xmx%JAVA_XMX_VALUE% -XX:+HeapDumpOnOutOfMemoryError -XX:HeapDumpPath="%GW_HOME%\heap-dump.hprof"
```

Comment:
This will use heap dump path with single quotes. @Praminda hope that would be okay?